tests/python/contrib/test_mlas.py

```diff
@@ -0,0 +1,324 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We should skip the tests if mlas is not enabled. We can add an if statement like below to the beginning of each `test_` function.

```
if get_global_func("tvm.contrib.mlas.batch_sgemm", allow_missing=True):
  print("skip because mlas is not enabled...")
  return
```

Reply:
Good suggestion, added.